fineract-investor/src/main/java/org/apache/fineract/investor/exception/ExternalAssetOwnerLoanProductAttributeAlreadyExistsException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.investor.exception;
+
+import org.apache.fineract.infrastructure.core.exception.AbstractPlatformDomainRuleException;
+import org.springframework.http.HttpStatus;
+
+public class ExternalAssetOwnerLoanProductAttributeAlreadyExistsException extends AbstractPlatformDomainRuleException {
+
+    private static final String errorCode = "error.msg.externalAssetOwnerLoanProductAttribute.exists";
+
+    public ExternalAssetOwnerLoanProductAttributeAlreadyExistsException(String errorMessage) {
+        super(errorCode, errorMessage, HttpStatus.CONFLICT);
```

Comment:
For domain rule violation we usually use 403. I think 409 we are using something really different.

Reply:
done.